batch7crud-roll4/src/main/webapp/resources/css/style.css

```diff
@@ -0,0 +1,203 @@
+body {
+    background: #ECF0F5;
+}
+
+a {
+    text-decoration: none;
+    font-weight: bold;
+    color: #3C8DCA;
+}
+
+a:hover {
+    color: #72AFD2;
+}
+
+.logo a {
+    color: #fff;
+    font-size: 26px;
+    font-weight: bold;
+    text-decoration: none;
+    display: block;
+}
+
+.logo a:hover {
+    background: #367FAA;
+}
+
+.username {
+    float: left;
+    display: block;
+    padding: 0 10px;
+    color: #fff;
+    font-size: 14px;
+    text-decoration: none;
+    line-height: 50px;
+}
+
+.username:hover {
+    background: #367FAA;
+}
+
+.logout-btn {
+    float: left;
+    padding: 15px 15px;
+}
+
+.logout-btn:hover {
+    float: left;
+    background: #367FAA;
+}
+
+/*navigation style*/
+.navigation ul {
+    list-style: none;
+}
+
+.navigation ul li {
+    display: block;
+    line-height: 50px;
+}
+
+.navigation ul li a {
+    display: block;
+    text-decoration: none;
+    font-weight: bold;
+    color: #444444;
+    font-size: 14px;
+    padding-left: 50px;
+}
+
+.navigation ul li:hover {
+    background-color: #F4F4F5;
+}
+
+li.employees {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the name of this overqualified element [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aoverqualified-elements)